openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -455,7 +439,7 @@ def post(self, request):
                         default=settings.LANGUAGE_CODE
                     )
                     notification = DeletionNotificationMessage().personalize(
-                        recipient=Recipient(lms_user_id=0, email_address=user_email),
+                        recipient=Recipient(request.user.id, email_address=user_email),
```

Comment:
This change puts conflicting information in the `Recipient` object, with a lms_user_id of the service user and a user_email of the retired user. I haven't discovered what the effect of this change will be - but I think the line should be left with a user_id of 0.

Reply:
Sorry about that. It was a mix-up during a rebase. Fixed it now.